### DISCUSSION TITLE
How to debug slow typescript-language-server?

### DISCUSSION THREAD

**Discussion by baptisteArno** [zed-industries/zed#8654] [Feature Requests]
The ts server is sometimes really slow:

- Error highlights take a lot of time to disapear
- It does not offer the import option right away on some objects I just wrote

I wanted to know what's the best way to debug that

**Comment by SomeoneToIgnore** [comment]
* there's "open language server logs" command that allows to see server logs and LSP json request/response logs for a certain server:
![image](https://github.com/zed-industries/zed/assets/2690773/2683f57d-a427-4e7f-bd95-a5c8eaccd26e)

* Zed logs might sometimes show noticeable troubles, ERROR messages, etc. related to the server
They are located in `~/Library/Logs/Zed/Zed.log` and there's also a command in Zed to display the latest ones

* general LSP server process CPU and memory monitoring might help
